src/pages/en/legal/modmail-privacy-policy.mdx

```diff
@@ -5,69 +5,67 @@ description: The privacy policy for the ModMail Discord bot and web interface.
 layout: /src/layouts/Page.astro
 ---
 
-# Foreword
+## Foreword
 
 The following document was created as required by the European Union General Data Protection Regulation (EU GDPR).
 You can find more information about that regulation on [gdpr.eu](https://gdpr.eu/).
 
-Our ModMail+ instance is contracted to Starchild Systems.
-This document details what and why data is collected by Starchild Systems for QuiltMC,
-how it is used and how to use your rights related to your data.
+This document details what and why data is collected by The Quilt Project through the our Modmail bot, how it is used and how to use your rights related to your data.
 
-The Quilt Project ("Us", "We", "Our") is the data controller for the data collected by Starchild Systems for QuiltMC.
+The Quilt Project ("Us", "We", "Our") is the data controller for the data collected through our Modmail Discord bot.
 
-# Data Collected
+## Data Collected
 
-## Discord Bot
+### Discord Bot
 
-We collect message content and attachments sent to the ModMail+ instance through Direct Message or by typing inside the thread channel.
+We collect message content and attachments sent to the ModMail bot through Direct Message or by typing inside the thread channel.
 
 We also collect the following information about the sender:
 * Discord ID
 * Discord Username
-* Discord Discriminator
+* Discord Discriminator (if applicable)
 * Discord Avatar URL
 
-We collect that data to preserve historical data and conduct moderation. The data is persisted across leaving the guild and is essential to the operation of the service.
+This data is collected to preserve historical records and conduct moderation. The data is persisted across leaving the guild and is essential to the operation of the service.
```

Comment:
If it isn't you, who is? This change is unclear.

Reply:
You are right, this change does seem unnecessary. I have reasons to believe it would not legally harm us (it can be assumed that we are the ones doing the moderation by default), but I went ahead and reverted it for clarity.